base_synchro/wizard/base_synchro.py

```diff
@@ -67,7 +66,8 @@ class base_synchro(osv.TransientModel):
     report_create = 0
     report_write = 0
 
-    def input(self, cr, uid, ids, value, context=None):
+    @api.model
+    def input(self, ids, value):
```

Comment:
It seems there is no need of input() method as there is no code inside it.
So remove it or comment it for future use / reference.